source/_components/sensor.filter.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: "Filter"
+description: "Instructions how to integrate Filter Sensors into Home Assistant."
+date: 2018-02-20
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Sensor
+ha_release: 0.12
+ha_iot_class: "Local Push"
+logo: home-assistant.png
+---
+
+The `filter` component supports sensors which process the **numeric states** from other entities. `filter` will apply a signal processing algorithm to a serie of states and produce a `new state` given the choosen algorithm.
```

Comment:
series
produces/ generates/ calculates

Reply:
How about:

```
`filter` will apply a signal processing algorithm to a series of states and generate a `new state` given the chosen algorithm.
```

I changed to *series* (plural), but each filter works with a single serie (singular). I'm no english speaking native, but it made sense to me. Tell me how to proceed.